### ISSUE TITLE
loss collapse

### ISSUE THREAD

**Issue by unfair67** [xieyujia/IPOT#2]
In pytorch demo, it usually collapses when using the algorithm. Maybe the loss function is different? I use the C matraix as the loss function, but it seems likes failed. How should I make it?
